src/snapshots/003.txt

```diff
@@ -1,14 +1,20 @@
 ~~~META
 description=Import statement
 ~~~SOURCE
-module [decoder]
+
+modAle
+
+[decoder]
 
 import json.Json
 
 decoder = Utf8.decode
 ~~~PARSE
 (file
-    (header 'decoder')
+    (malformed 'missing_header')
+    (ident '' 'modAle')
+    (list
+        (ident '' 'decoder'))
```

Comment:
It's recognising the missing header... but then it continues to parse the rest of that line as an ident and list. Not sure the best way to deal with this. 

@gamebox I'll probably need your guidance here to make much progress. I feel like I'm poking around in the dark a little.